storage/merkle/node.go

```diff
@@ -3,23 +3,110 @@
 package merkle
 
 import (
-	"github.com/onflow/flow-go/crypto/hash"
+	"golang.org/x/crypto/blake2b"
```

Comment:
hmm, this seems like one of the changes that needs @tarakby's approval.

Reply:
Blake2 was already used by this package, there is nothing new. The import update is a result of a file refactor, so all good